cpp/Platform.Equality.Tests/Platform.Equality.Benchmark.cpp

```diff
@@ -0,0 +1,127 @@
+#include <Platform.Equality.h>
+#include <benchmark/benchmark.h>
+
+#include <ctime>
+#include <iostream>
+#include <span>
+#include <any>
+
+namespace Platform::Equality::Benchmark
```

Comment:
Please rename to `Platform::Equality::Benchmarks` (`s` added at the end). And move to separate folder `Platform.Equality.Benchmarks`. Benchmarks are not Tests.